src/jiran.js

```diff
@@ -146,4 +146,3 @@ program.parse(process.argv)
 if (program.args.length === 0) {
   program.help()
 }
-
```

Comment:
correct, I am suing `standard` for coding style and new line at the end of a file is required